src/ctap/crypto_wrapper.rs

```diff
@@ -42,6 +44,12 @@ pub const ECDSA_CREDENTIAL_ID_SIZE: usize = 113;
 // See encrypt_key_handle v1 documentation.
 pub const MAX_CREDENTIAL_ID_SIZE: usize = 113;
 
+const ECDSA_CREDENTIAL_ID_VERSION: u8 = 0x01;
+#[cfg(feature = "ed25519")]
```

Comment:
@ia0 my main concern about this piece of code is a potential "re-use" of the constant `2` depending on enabled features. Isn't that safer to either prefix it with underscore (as it may be unused depending on compilation features) or just allow unused on the constant instead of protecting it with a feature?

Reply:
Good point. Yes, I think we should definitely have those constants in a global/eternal namespace (i.e. not even reuse deprecated ones).

So concretely, let's just replace the `#[cfg(feature = "ed25519")]` with `#[allow(dead_code)]` (or equivalent).

Reply:
Mmm... actually, I don't think that enough to be really foolproof. We need to use an enum:

```rust
#[repr(u8)]
enum CredentialIdVersion {
    Ecdsa = 0x01,
    #[allow(dead_code)]
    Ed25519 = 0x02,
}
```

Reply:
Done